app/templates/_karma.conf.js

```diff
@@ -17,7 +17,10 @@ module.exports = function (config) {
 			'app/app.js',
 			'.generated/templates.js'<% if (useRouting) { %>,
 			'.generated/app/modules/routes/routes.concat.js'<% } %>,
-			'.generated/app/**/*.concat.js' // Leave this last!!!
+			'app/modules/**/*.module.js',
```

Comment:
In all my testing I've never had a single issue with that. But if you feel that's important I can try to make the other way work.